static/js/hashchange.js

```diff
@@ -303,6 +303,10 @@ exports.go_to_location = function (hash) {
     window.location.hash = hash;
 };
 
+exports.get_current_hash = function () {
+    return window.location.hash;
+};
```

Comment:
Rather than exporting this sort of function, I was imagining something more like a pair of functions used by `reload.js` for encoding/decoding this feature.  As it is, we're not actually transferring control over logic that deals with hashes between modules.